scripts/weather/WeatherPreview.js

```diff
@@ -57,15 +59,20 @@ const fiveDayWeather = (weather) => {
 
 const dailyWeather = (day, weather) => {
   const dayString = day.toString()
+  const weekdayNumber = new Date(weather.daily[dayString].dt*1000).getDay()
+  const weekDayArray = [ 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday' ]
+  const weekDayString = weekDayArray[weekdayNumber]
   const imagePath = `../../weatherImages/${weather.daily[dayString].weather["0"].icon}.png`
```

Comment:
Just caught this now, but since this `imagePath` is ultimately just going to be rendered in HTML, isn't the path really just relative from `index.html`, and not this file? So, the path could actually just be `./weatherImages/(et cetera)`, and _maybe_ is worth changing to that? I assume it is working now just because `index.html` is in the root of our web server so going up a directory doesn't mean anything maybe?

Reply:
Good point! I forgot that the image path is relative to the `index.html`